lte/gateway/deploy/agw_install.sh

```diff
@@ -58,6 +67,21 @@ if [ "$(uname -r)" != "4.9.0-9-amd64" ]; then
   apt update
   # Installing prerequesites, Kvers, headers
   apt install -y sudo python-minimal aptitude linux-image-4.9.0-9-amd64 linux-headers-4.9.0-9-amd64
```

Comment:
If headers-4.9.0-9 is no longer upstream, let's remove it from this line.

Reply:
Done!